DEBUG - 2025-06-24 23:04:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-24 23:05:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-24 23:06:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-24 23:08:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-24 23:08:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-24 23:08:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-24 23:09:03 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-24 23:20:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-24 23:20:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-24 23:22:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-24 23:25:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-24 23:25:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-06-24 23:26:06 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'tienda-abierta' in C:\xampp\htdocs\ProT3_46516214\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\ProT3_46516214\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'tienda-abierta', 3306, '', 0)
#1 C:\xampp\htdocs\ProT3_46516214\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\ProT3_46516214\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\ProT3_46516214\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ProT3_46516214\system\Validation\Rules.php(175): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ProT3_46516214\system\Validation\StrictRules\Rules.php(187): CodeIgniter\Validation\Rules->is_unique('Matias100605', 'user.nickname', Array)
#6 C:\xampp\htdocs\ProT3_46516214\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('Matias100605', 'user.nickname', Array, NULL, 'nickname')
#7 C:\xampp\htdocs\ProT3_46516214\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('nickname', 'nickname', 'Matias100605', Array, Array, 'nickname')
#8 C:\xampp\htdocs\ProT3_46516214\system\Controller.php(138): CodeIgniter\Validation\Validation->run()
#9 C:\xampp\htdocs\ProT3_46516214\app\Controllers\UserController.php(22): CodeIgniter\Controller->validate(Array)
#10 C:\xampp\htdocs\ProT3_46516214\system\CodeIgniter.php(933): App\Controllers\UserController->formValidation()
#11 C:\xampp\htdocs\ProT3_46516214\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#12 C:\xampp\htdocs\ProT3_46516214\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\ProT3_46516214\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\ProT3_46516214\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#15 C:\xampp\htdocs\ProT3_46516214\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#16 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'tienda-abierta' in C:\xampp\htdocs\ProT3_46516214\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\ProT3_46516214\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\ProT3_46516214\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\ProT3_46516214\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\xampp\htdocs\ProT3_46516214\system\Validation\Rules.php(175): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\ProT3_46516214\system\Validation\StrictRules\Rules.php(187): CodeIgniter\Validation\Rules->is_unique('Matias100605', 'user.nickname', Array)
#5 C:\xampp\htdocs\ProT3_46516214\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('Matias100605', 'user.nickname', Array, NULL, 'nickname')
#6 C:\xampp\htdocs\ProT3_46516214\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('nickname', 'nickname', 'Matias100605', Array, Array, 'nickname')
#7 C:\xampp\htdocs\ProT3_46516214\system\Controller.php(138): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\ProT3_46516214\app\Controllers\UserController.php(22): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\ProT3_46516214\system\CodeIgniter.php(933): App\Controllers\UserController->formValidation()
#10 C:\xampp\htdocs\ProT3_46516214\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#11 C:\xampp\htdocs\ProT3_46516214\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ProT3_46516214\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\ProT3_46516214\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\ProT3_46516214\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2025-06-24 23:26:06 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'tienda-abierta'
[Method: POST, Route: enviar-form]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `user`
WHERE `nickname` = :nickname:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\Rules.php(175): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\StrictRules\Rules.php(187): CodeIgniter\Validation\Rules->is_unique('Matias100605', 'user.nickname', [...])
 5 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('Matias100605', 'user.nickname', [...], null, 'nickname')
 6 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('nickname', 'nickname', 'Matias100605', [...], [...], 'nickname')
 7 SYSTEMPATH\Controller.php(138): CodeIgniter\Validation\Validation->run()
 8 APPPATH\Controllers\UserController.php(22): CodeIgniter\Controller->validate([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->formValidation()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-06-24 23:30:54 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'tienda-abierta' in C:\xampp\htdocs\ProT3_46516214\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\ProT3_46516214\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'tienda-abierta', 3306, '', 0)
#1 C:\xampp\htdocs\ProT3_46516214\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\ProT3_46516214\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\ProT3_46516214\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ProT3_46516214\system\Validation\Rules.php(175): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ProT3_46516214\system\Validation\StrictRules\Rules.php(187): CodeIgniter\Validation\Rules->is_unique('Matias100605', 'user.nickname', Array)
#6 C:\xampp\htdocs\ProT3_46516214\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('Matias100605', 'user.nickname', Array, NULL, 'nickname')
#7 C:\xampp\htdocs\ProT3_46516214\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('nickname', 'nickname', 'Matias100605', Array, Array, 'nickname')
#8 C:\xampp\htdocs\ProT3_46516214\system\Controller.php(138): CodeIgniter\Validation\Validation->run()
#9 C:\xampp\htdocs\ProT3_46516214\app\Controllers\UserController.php(22): CodeIgniter\Controller->validate(Array)
#10 C:\xampp\htdocs\ProT3_46516214\system\CodeIgniter.php(933): App\Controllers\UserController->formValidation()
#11 C:\xampp\htdocs\ProT3_46516214\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#12 C:\xampp\htdocs\ProT3_46516214\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\ProT3_46516214\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\ProT3_46516214\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#15 C:\xampp\htdocs\ProT3_46516214\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#16 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'tienda-abierta' in C:\xampp\htdocs\ProT3_46516214\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\ProT3_46516214\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\ProT3_46516214\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\ProT3_46516214\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\xampp\htdocs\ProT3_46516214\system\Validation\Rules.php(175): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\ProT3_46516214\system\Validation\StrictRules\Rules.php(187): CodeIgniter\Validation\Rules->is_unique('Matias100605', 'user.nickname', Array)
#5 C:\xampp\htdocs\ProT3_46516214\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('Matias100605', 'user.nickname', Array, NULL, 'nickname')
#6 C:\xampp\htdocs\ProT3_46516214\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('nickname', 'nickname', 'Matias100605', Array, Array, 'nickname')
#7 C:\xampp\htdocs\ProT3_46516214\system\Controller.php(138): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\ProT3_46516214\app\Controllers\UserController.php(22): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\ProT3_46516214\system\CodeIgniter.php(933): App\Controllers\UserController->formValidation()
#10 C:\xampp\htdocs\ProT3_46516214\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#11 C:\xampp\htdocs\ProT3_46516214\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ProT3_46516214\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\ProT3_46516214\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\ProT3_46516214\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2025-06-24 23:30:54 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'tienda-abierta'
[Method: POST, Route: enviar-form]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `user`
WHERE `nickname` = :nickname:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\Rules.php(175): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\StrictRules\Rules.php(187): CodeIgniter\Validation\Rules->is_unique('Matias100605', 'user.nickname', [...])
 5 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('Matias100605', 'user.nickname', [...], null, 'nickname')
 6 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('nickname', 'nickname', 'Matias100605', [...], [...], 'nickname')
 7 SYSTEMPATH\Controller.php(138): CodeIgniter\Validation\Validation->run()
 8 APPPATH\Controllers\UserController.php(22): CodeIgniter\Controller->validate([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->formValidation()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-06-24 23:32:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-24 23:32:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-24 23:33:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-06-24 23:34:23 --> mysqli_sql_exception: Duplicate entry '' for key 'perfil_id' in C:\xampp\htdocs\ProT3_46516214\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ProT3_46516214\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ProT3_46516214\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ProT3_46516214\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ProT3_46516214\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\ProT3_46516214\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ProT3_46516214\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ProT3_46516214\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\ProT3_46516214\system\BaseModel.php(754): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\ProT3_46516214\app\Controllers\UserController.php(39): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\ProT3_46516214\system\CodeIgniter.php(933): App\Controllers\UserController->formValidation()
#10 C:\xampp\htdocs\ProT3_46516214\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#11 C:\xampp\htdocs\ProT3_46516214\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ProT3_46516214\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\ProT3_46516214\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\ProT3_46516214\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2025-06-24 23:34:23 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'perfil_id'
[Method: POST, Route: enviar-form]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (:name:, :last_name:, :nickname:, :email:, :password:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(754): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\UserController.php(39): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->formValidation()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-24 23:34:23 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'perfil_id'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (\'asdfasdfasdggqw\', \'asdfasdf\', \'Matias1006asdf\', \'matias100605+1@gmail.com\', \'$2y$10$r72I8h0DFynNyHA3z.sriunLStotXXexWR67bQKKB2DIlechiJ3IW\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (\'asdfasdfasdggqw\', \'asdfasdf\', \'Matias1006asdf\', \'matias100605+1@gmail.com\', \'$2y$10$r72I8h0DFynNyHA3z.sriunLStotXXexWR67bQKKB2DIlechiJ3IW\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (:name:, :last_name:, :nickname:, :email:, :password:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(754): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\UserController.php(39): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->formValidation()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-24 23:34:23 --> [Caused by] mysqli_sql_exception: Duplicate entry '' for key 'perfil_id'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (\'asdfasdfasdggqw\', \'asdfasdf\', \'Matias1006asdf\', \'matias100605+1@gmail.com\', \'$2y$10$r72I8h0DFynNyHA3z.sriunLStotXXexWR67bQKKB2DIlechiJ3IW\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (\'asdfasdfasdggqw\', \'asdfasdf\', \'Matias1006asdf\', \'matias100605+1@gmail.com\', \'$2y$10$r72I8h0DFynNyHA3z.sriunLStotXXexWR67bQKKB2DIlechiJ3IW\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (\'asdfasdfasdggqw\', \'asdfasdf\', \'Matias1006asdf\', \'matias100605+1@gmail.com\', \'$2y$10$r72I8h0DFynNyHA3z.sriunLStotXXexWR67bQKKB2DIlechiJ3IW\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (:name:, :last_name:, :nickname:, :email:, :password:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(754): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\UserController.php(39): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->formValidation()
11 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
12 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-06-24 23:34:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-06-24 23:35:20 --> mysqli_sql_exception: Duplicate entry '' for key 'perfil_id' in C:\xampp\htdocs\ProT3_46516214\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ProT3_46516214\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ProT3_46516214\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ProT3_46516214\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ProT3_46516214\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\ProT3_46516214\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ProT3_46516214\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ProT3_46516214\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\ProT3_46516214\system\BaseModel.php(754): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\ProT3_46516214\app\Controllers\UserController.php(39): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\ProT3_46516214\system\CodeIgniter.php(933): App\Controllers\UserController->formValidation()
#10 C:\xampp\htdocs\ProT3_46516214\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#11 C:\xampp\htdocs\ProT3_46516214\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ProT3_46516214\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\ProT3_46516214\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\ProT3_46516214\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2025-06-24 23:35:20 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'perfil_id'
[Method: POST, Route: enviar-form]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (:name:, :last_name:, :nickname:, :email:, :password:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(754): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\UserController.php(39): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->formValidation()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-24 23:35:20 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'perfil_id'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (\'asdfasdfasdggqw\', \'asdfasdf\', \'asdfasdgqwgq\', \'askfkdljlkjadalkjs@jasdkfj.com\', \'$2y$10$LBBgytgdAjk5t4nzDgHEgOJDEY5HLhAtpnbzpP2GAUbbS5wWrXMAS\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (\'asdfasdfasdggqw\', \'asdfasdf\', \'asdfasdgqwgq\', \'askfkdljlkjadalkjs@jasdkfj.com\', \'$2y$10$LBBgytgdAjk5t4nzDgHEgOJDEY5HLhAtpnbzpP2GAUbbS5wWrXMAS\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (:name:, :last_name:, :nickname:, :email:, :password:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(754): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\UserController.php(39): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->formValidation()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-24 23:35:20 --> [Caused by] mysqli_sql_exception: Duplicate entry '' for key 'perfil_id'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (\'asdfasdfasdggqw\', \'asdfasdf\', \'asdfasdgqwgq\', \'askfkdljlkjadalkjs@jasdkfj.com\', \'$2y$10$LBBgytgdAjk5t4nzDgHEgOJDEY5HLhAtpnbzpP2GAUbbS5wWrXMAS\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (\'asdfasdfasdggqw\', \'asdfasdf\', \'asdfasdgqwgq\', \'askfkdljlkjadalkjs@jasdkfj.com\', \'$2y$10$LBBgytgdAjk5t4nzDgHEgOJDEY5HLhAtpnbzpP2GAUbbS5wWrXMAS\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (\'asdfasdfasdggqw\', \'asdfasdf\', \'asdfasdgqwgq\', \'askfkdljlkjadalkjs@jasdkfj.com\', \'$2y$10$LBBgytgdAjk5t4nzDgHEgOJDEY5HLhAtpnbzpP2GAUbbS5wWrXMAS\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (:name:, :last_name:, :nickname:, :email:, :password:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(754): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\UserController.php(39): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->formValidation()
11 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
12 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-06-24 23:36:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-24 23:36:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-24 23:36:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-06-24 23:37:33 --> mysqli_sql_exception: Duplicate entry '' for key 'perfil_id' in C:\xampp\htdocs\ProT3_46516214\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ProT3_46516214\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ProT3_46516214\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ProT3_46516214\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ProT3_46516214\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\ProT3_46516214\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ProT3_46516214\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ProT3_46516214\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\ProT3_46516214\system\BaseModel.php(754): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\ProT3_46516214\app\Controllers\UserController.php(39): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\ProT3_46516214\system\CodeIgniter.php(933): App\Controllers\UserController->formValidation()
#10 C:\xampp\htdocs\ProT3_46516214\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#11 C:\xampp\htdocs\ProT3_46516214\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ProT3_46516214\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\ProT3_46516214\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\ProT3_46516214\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2025-06-24 23:37:33 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'perfil_id'
[Method: POST, Route: enviar-form]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (:name:, :last_name:, :nickname:, :email:, :password:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(754): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\UserController.php(39): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->formValidation()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-24 23:37:33 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'perfil_id'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (\'alan\', \'baez aranda\', \'Fatiga100\', \'baalan@gmail.com\', \'$2y$10$hMsRnc1BVOT8AjCtDRTMgOpEGIsSueyjt48TVeXqT3e9zrdr75x9.\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (\'alan\', \'baez aranda\', \'Fatiga100\', \'baalan@gmail.com\', \'$2y$10$hMsRnc1BVOT8AjCtDRTMgOpEGIsSueyjt48TVeXqT3e9zrdr75x9.\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (:name:, :last_name:, :nickname:, :email:, :password:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(754): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\UserController.php(39): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->formValidation()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-24 23:37:33 --> [Caused by] mysqli_sql_exception: Duplicate entry '' for key 'perfil_id'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (\'alan\', \'baez aranda\', \'Fatiga100\', \'baalan@gmail.com\', \'$2y$10$hMsRnc1BVOT8AjCtDRTMgOpEGIsSueyjt48TVeXqT3e9zrdr75x9.\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (\'alan\', \'baez aranda\', \'Fatiga100\', \'baalan@gmail.com\', \'$2y$10$hMsRnc1BVOT8AjCtDRTMgOpEGIsSueyjt48TVeXqT3e9zrdr75x9.\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (\'alan\', \'baez aranda\', \'Fatiga100\', \'baalan@gmail.com\', \'$2y$10$hMsRnc1BVOT8AjCtDRTMgOpEGIsSueyjt48TVeXqT3e9zrdr75x9.\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (:name:, :last_name:, :nickname:, :email:, :password:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(754): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\UserController.php(39): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->formValidation()
11 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
12 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-06-24 23:38:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-06-24 23:39:39 --> mysqli_sql_exception: Duplicate entry '' for key 'perfil_id' in C:\xampp\htdocs\ProT3_46516214\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ProT3_46516214\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ProT3_46516214\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ProT3_46516214\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ProT3_46516214\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\ProT3_46516214\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ProT3_46516214\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ProT3_46516214\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\ProT3_46516214\system\BaseModel.php(754): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\ProT3_46516214\app\Controllers\UserController.php(39): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\ProT3_46516214\system\CodeIgniter.php(933): App\Controllers\UserController->formValidation()
#10 C:\xampp\htdocs\ProT3_46516214\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#11 C:\xampp\htdocs\ProT3_46516214\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ProT3_46516214\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\ProT3_46516214\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\ProT3_46516214\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2025-06-24 23:39:39 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'perfil_id'
[Method: POST, Route: enviar-form]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (:name:, :last_name:, :nickname:, :email:, :password:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(754): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\UserController.php(39): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->formValidation()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-24 23:39:39 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'perfil_id'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (\'Alan\', \'Baez Arand\', \'Fatiga100\', \'AlanBA@gmail.com\', \'$2y$10$.rDNm95EoOj6Q8Iq77IOYebFWDQobf1oNTBDk7KUEHRXKSoo7au6G\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (\'Alan\', \'Baez Arand\', \'Fatiga100\', \'AlanBA@gmail.com\', \'$2y$10$.rDNm95EoOj6Q8Iq77IOYebFWDQobf1oNTBDk7KUEHRXKSoo7au6G\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (:name:, :last_name:, :nickname:, :email:, :password:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(754): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\UserController.php(39): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->formValidation()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-24 23:39:39 --> [Caused by] mysqli_sql_exception: Duplicate entry '' for key 'perfil_id'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (\'Alan\', \'Baez Arand\', \'Fatiga100\', \'AlanBA@gmail.com\', \'$2y$10$.rDNm95EoOj6Q8Iq77IOYebFWDQobf1oNTBDk7KUEHRXKSoo7au6G\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (\'Alan\', \'Baez Arand\', \'Fatiga100\', \'AlanBA@gmail.com\', \'$2y$10$.rDNm95EoOj6Q8Iq77IOYebFWDQobf1oNTBDk7KUEHRXKSoo7au6G\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (\'Alan\', \'Baez Arand\', \'Fatiga100\', \'AlanBA@gmail.com\', \'$2y$10$.rDNm95EoOj6Q8Iq77IOYebFWDQobf1oNTBDk7KUEHRXKSoo7au6G\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (:name:, :last_name:, :nickname:, :email:, :password:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(754): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\UserController.php(39): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->formValidation()
11 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
12 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-06-24 23:42:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-24 23:42:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-24 23:42:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-24 23:42:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-06-24 23:43:19 --> mysqli_sql_exception: Duplicate entry '' for key 'perfil_id' in C:\xampp\htdocs\ProT3_46516214\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ProT3_46516214\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ProT3_46516214\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ProT3_46516214\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ProT3_46516214\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\ProT3_46516214\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ProT3_46516214\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ProT3_46516214\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\ProT3_46516214\system\BaseModel.php(754): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\ProT3_46516214\app\Controllers\UserController.php(39): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\ProT3_46516214\system\CodeIgniter.php(933): App\Controllers\UserController->formValidation()
#10 C:\xampp\htdocs\ProT3_46516214\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#11 C:\xampp\htdocs\ProT3_46516214\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ProT3_46516214\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\ProT3_46516214\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\ProT3_46516214\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2025-06-24 23:43:19 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'perfil_id'
[Method: POST, Route: enviar-form]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (:name:, :last_name:, :nickname:, :email:, :password:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(754): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\UserController.php(39): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->formValidation()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-24 23:43:19 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'perfil_id'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (\'Matias\', \'Solis Schneeberger\', \'Matias1006asdf\', \'matias.solis.sch@gmail.com\', \'$2y$10$tb4SAx4.zVFIrQ7xShx7W.NNJwwnJOuY6s/QNYT5sanKRWIC3laJ2\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (\'Matias\', \'Solis Schneeberger\', \'Matias1006asdf\', \'matias.solis.sch@gmail.com\', \'$2y$10$tb4SAx4.zVFIrQ7xShx7W.NNJwwnJOuY6s/QNYT5sanKRWIC3laJ2\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (:name:, :last_name:, :nickname:, :email:, :password:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(754): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\UserController.php(39): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->formValidation()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-24 23:43:19 --> [Caused by] mysqli_sql_exception: Duplicate entry '' for key 'perfil_id'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (\'Matias\', \'Solis Schneeberger\', \'Matias1006asdf\', \'matias.solis.sch@gmail.com\', \'$2y$10$tb4SAx4.zVFIrQ7xShx7W.NNJwwnJOuY6s/QNYT5sanKRWIC3laJ2\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (\'Matias\', \'Solis Schneeberger\', \'Matias1006asdf\', \'matias.solis.sch@gmail.com\', \'$2y$10$tb4SAx4.zVFIrQ7xShx7W.NNJwwnJOuY6s/QNYT5sanKRWIC3laJ2\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (\'Matias\', \'Solis Schneeberger\', \'Matias1006asdf\', \'matias.solis.sch@gmail.com\', \'$2y$10$tb4SAx4.zVFIrQ7xShx7W.NNJwwnJOuY6s/QNYT5sanKRWIC3laJ2\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (:name:, :last_name:, :nickname:, :email:, :password:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(754): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\UserController.php(39): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->formValidation()
11 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
12 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-06-24 23:47:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-24 23:47:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-06-24 23:47:53 --> mysqli_sql_exception: Duplicate entry '2' for key 'perfil_id' in C:\xampp\htdocs\ProT3_46516214\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ProT3_46516214\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ProT3_46516214\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ProT3_46516214\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ProT3_46516214\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\ProT3_46516214\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ProT3_46516214\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ProT3_46516214\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\ProT3_46516214\system\BaseModel.php(754): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\ProT3_46516214\app\Controllers\UserController.php(39): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\ProT3_46516214\system\CodeIgniter.php(933): App\Controllers\UserController->formValidation()
#10 C:\xampp\htdocs\ProT3_46516214\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#11 C:\xampp\htdocs\ProT3_46516214\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ProT3_46516214\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\ProT3_46516214\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\ProT3_46516214\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2025-06-24 23:47:53 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '2' for key 'perfil_id'
[Method: POST, Route: enviar-form]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (:name:, :last_name:, :nickname:, :email:, :password:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(754): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\UserController.php(39): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->formValidation()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-24 23:47:53 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '2' for key 'perfil_id'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (\'Alan\', \'Baez Arand\', \'Fatiga100\', \'baalan@gmail.com\', \'$2y$10$kF5GjLZH58tEQ4n8hvEsQecx1o.yyROfRVXWP0SohHj29VDXN0rBC\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (\'Alan\', \'Baez Arand\', \'Fatiga100\', \'baalan@gmail.com\', \'$2y$10$kF5GjLZH58tEQ4n8hvEsQecx1o.yyROfRVXWP0SohHj29VDXN0rBC\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (:name:, :last_name:, :nickname:, :email:, :password:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(754): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\UserController.php(39): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->formValidation()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-24 23:47:53 --> [Caused by] mysqli_sql_exception: Duplicate entry '2' for key 'perfil_id'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (\'Alan\', \'Baez Arand\', \'Fatiga100\', \'baalan@gmail.com\', \'$2y$10$kF5GjLZH58tEQ4n8hvEsQecx1o.yyROfRVXWP0SohHj29VDXN0rBC\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (\'Alan\', \'Baez Arand\', \'Fatiga100\', \'baalan@gmail.com\', \'$2y$10$kF5GjLZH58tEQ4n8hvEsQecx1o.yyROfRVXWP0SohHj29VDXN0rBC\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (\'Alan\', \'Baez Arand\', \'Fatiga100\', \'baalan@gmail.com\', \'$2y$10$kF5GjLZH58tEQ4n8hvEsQecx1o.yyROfRVXWP0SohHj29VDXN0rBC\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`name`, `last_name`, `nickname`, `email`, `password`) VALUES (:name:, :last_name:, :nickname:, :email:, :password:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(754): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\UserController.php(39): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UserController->formValidation()
11 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
12 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-06-24 23:51:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-24 23:51:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-24 23:58:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
